[2022-04-28 07:25:47,488] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first.save scheduled__2022-04-27T00:00:00+00:00 [queued]>
[2022-04-28 07:25:47,510] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first.save scheduled__2022-04-27T00:00:00+00:00 [queued]>
[2022-04-28 07:25:47,511] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 07:25:47,511] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-28 07:25:47,512] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-28 07:25:47,552] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): save> on 2022-04-27 00:00:00+00:00
[2022-04-28 07:25:47,573] {standard_task_runner.py:52} INFO - Started process 10198 to run task
[2022-04-28 07:25:47,590] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first', 'save', 'scheduled__2022-04-27T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/first.py', '--cfg-path', '/tmp/tmp4zmpd5by', '--error-file', '/tmp/tmphadvh57y']
[2022-04-28 07:25:47,615] {standard_task_runner.py:80} INFO - Job 14: Subtask save
[2022-04-28 07:25:47,915] {logging_mixin.py:109} INFO - Running <TaskInstance: first.save scheduled__2022-04-27T00:00:00+00:00 [running]> on host 65467f9f0591
[2022-04-28 07:25:48,346] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first
AIRFLOW_CTX_TASK_ID=save
AIRFLOW_CTX_EXECUTION_DATE=2022-04-27T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-27T00:00:00+00:00
[2022-04-28 07:25:48,361] {first.py:28} INFO - performing saving
[2022-04-28 07:25:48,376] {python.py:175} INFO - Done. Returned value was: None
[2022-04-28 07:25:48,422] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=first, task_id=save, execution_date=20220427T000000, start_date=20220428T072547, end_date=20220428T072548
[2022-04-28 07:25:48,555] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-28 07:25:48,706] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
